### DISCUSSION TITLE
How do I sync two profiles?

### DISCUSSION THREAD

**Discussion by carluri** [phw198/OutlookGoogleCalendarSync#1995] [Q&A] [answered]
Hello,

I'm trying to sync two different outlook calendars to google calendar. I've created two profiles. Each of them independently sync well. However, I thought the sync utility was going to run automatically for the two profiles, but it seems to work only with the one I have selected. Is this how is intended? or how do I get it to work the way I want it?

thanks for the help

**Comment by phw198** [comment, marked as answer]
Just set a sync interval in each Profile.
